teammapper-backend/src/map/entities/mmpNode.entity.ts

```diff
@@ -17,106 +17,100 @@ import { validateOrReject, IsDefined } from 'class-validator';
-  nodeMap: MmpMap
-  /* eslint-enable @typescript-eslint/no-unused-vars */
+  nodeMap: MmpMap;
 
-  /* eslint-disable @typescript-eslint/no-unused-vars */
-  @ManyToOne((type) => MmpNode, (node) => node.children, {
+  @ManyToOne(() => MmpNode, (node) => node.children, {
     onDelete: 'CASCADE',
   })
   @JoinColumn([
     { name: 'nodeMapId', referencedColumnName: 'nodeMapId' },
     { name: 'nodeParentId', referencedColumnName: 'id' },
   ])
   @Index()
-  nodeParent: MmpNode
-  /* eslint-enable @typescript-eslint/no-unused-vars */
+  nodeParent: MmpNode;
 
-  /* eslint-disable @typescript-eslint/no-unused-vars */
-  @OneToMany((type) => MmpNode, (node) => node.nodeParent)
-  children: MmpNode[]
-  /* eslint-enable @typescript-eslint/no-unused-vars */
+  @OneToMany(() => MmpNode, (node) => node.nodeParent)
+  children: MmpNode[];
 
-  @Column({ default: false })
+  @Column({ type: 'boolean', default: false })
   @IsDefined()
-  root: boolean
+  root: boolean;
 
   @Column({ type: 'float' })
   @IsDefined()
-  coordinatesX: number
+  coordinatesX: number;
 
   @Column({ type: 'float' })
   @IsDefined()
-  coordinatesY: number
+  coordinatesY: number;
 
-  @Column({ nullable: true })
-  colorsName: string
+  @Column({ type: 'varchar', nullable: true })
+  colorsName: string | null;
 
-  @Column({ nullable: true })
-  colorsBackground: string
+  @Column({ type: 'varchar', nullable: true })
+  colorsBackground: string | null;
 
-  @Column({ nullable: true })
-  colorsBranch: string
+  @Column({ type: 'varchar', nullable: true })
+  colorsBranch: string | null;
 
-  @Column({ nullable: true })
-  fontSize: number
+  @Column({ type: 'integer', nullable: true })
+  fontSize: number | null;
 
-  @Column({ nullable: true })
-  fontStyle: string
+  @Column({ type: 'varchar', nullable: true })
+  fontStyle: string | null;
 
-  @Column({ nullable: true })
-  fontWeight: string
+  @Column({ type: 'varchar', nullable: true })
+  fontWeight: string | null;
 
-  @Column({ nullable: true })
-  imageSrc: string
+  @Column({ type: 'varchar', nullable: true })
+  imageSrc: string | null;
 
-  @Column({ nullable: true, default: 60 })
-  imageSize: number
+  @Column({ type: 'integer', nullable: true, default: 60 })
+  imageSize: number | null;
 
-  @Column({ nullable: true })
-  linkHref: string
+  @Column({ type: 'varchar', nullable: true })
+  linkHref: string | null;
 
-  @Column({ nullable: true })
-  locked: boolean
+  @Column({ type: 'boolean', nullable: true })
+  locked: boolean | null;
 
-  @Column({ default: false })
+  @Column({ type: 'boolean', default: false })
   @IsDefined()
-  detached: boolean
+  detached: boolean;
 
-  @Column({ nullable: true, type: 'float' })
-  k: number
+  @Column({ type: 'float', nullable: true })
+  k: number | null;
 
   @PrimaryColumn('uuid')
   @Index()
   @IsDefined()
-  nodeMapId: string
+  nodeMapId: string;
 
-  @Column({ nullable: true })
-  nodeParentId: string
+  @Column({ type: 'uuid', nullable: true })
```

Comment:
I would propose to use string here instead. https://orkhan.gitbook.io/typeorm/docs/entities#column-types-for-postgres

uuid is a column type in postgres and currently it is a string. This should trigger a database column type change, which is not what we want?